movie_api/movie_api/settings.py

```diff
@@ -33,14 +33,16 @@
     'django.middleware.common.CommonMiddleware',
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+    # 'django.middleware.csrf.CsrfViewMiddleware',
```

Comment:
👍 hee haw

FWIW we use `django-cors-headers` to let us specify CORS_ALLOWED_ORIGINS (aka whatever allowed client URLs we want), and blacklist the rest: https://stackoverflow.com/questions/66288845/csrf-in-local-development